specifications/sep-payment-channel-transactions.md

```diff
@@ -457,20 +484,33 @@ The participants:
 
 1. Increment i.
 2. I and R build the bump transaction B_i.
-3. Increment i.
-4. Sign and exchange a closing transaction C_i, that closes the channel with
-disbursement matching the most recent agreed state.
-5. Sign and exchange a declaration transaction D_i.
-6. I and R sign and exchange signatures for bump transaction B_i.
+3. Set e to i.
+4. I or R signs and shares signatures for:
+   - A declaration transaction D_i+1.
+   - A closing transaction C_i+1, that closes the channel with disbursement
+   matching the most recent agreed state.
+5. R or I signs and shares signatures for B_i, D_i+1, and C_i+1.
+6. Increment i.
 7. I or R submit B_i.
-9. Set e to B_i's iteration number.
 
 The transactions are constructed as follows:
 
 - B_i, the _bump transaction_, bumps the sequence number of escrow account E
 such that only the most recent transaction set is valid. B has source account
 EI, sequence number s_i.
 
+  B_i has two signers in its `extraSigners` precondition that ensures that if
+  B_i is authorized, signed, and submitted, that the signatures for D_i+1 and
+  C_i+1 are revealed. The signer is specified as:
+
+  - A ed25519 signed payload signer configured with:
+    - Payload set to the transaction hash of D_i.
+    - Public key set to R's signer.
```

Comment:
❓ same thing here, this should be `other participant's signer`?

Reply:
Other's participants here would be ambigious because both participants build the transaction and they need to use the same participant here. You're right though, it isn't always R here. I've updated this section in 8c35f26.